src/Infrastructure/Event/Workflow/WorkflowEventListener.php

```diff
@@ -7,10 +7,7 @@
 
 namespace SprykerSdk\Sdk\Infrastructure\Event\Workflow;
 
-use SprykerSdk\Sdk\Extension\Dependency\Event\WorkflowEventHandlerInterface;
-use SprykerSdk\Sdk\Extension\Dependency\Event\WorkflowEventInterface;
-use SprykerSdk\Sdk\Extension\Dependency\Event\WorkflowGuardEventHandlerInterface;
-use SprykerSdk\Sdk\Extension\Exception\InvalidServiceException;
+use SprykerSdk\Sdk\Infrastructure\Exception\InvalidServiceException;
```

Comment:
This listener is a completely application-layer service. The only thing that confuses here is `protected ?InputInterface $input = null;` I believe, it can be extracted to the separate service in the infra layer and provided as a dependency.